### PR TITLE
[install] use dedicated version file for released version

### PR DESCRIPTION
**Pull request by lispyclouds**

### CODE REVIEW

File: install [hunk 1]
```diff
@@ -58,7 +58,6 @@ do
             ;;
         *)  # unknown option
             print_help
-            shift
```
Comment (lispyclouds) [inline install:61]: removed this as `print_help` has an `exit 1` at the end and the `shift` is redundant.

**Comment by lispyclouds** [comment]
not sure if the release script needs to be updated

**Review by ericdallo — COMMENTED** [review]
@lispyclouds we still need to update that file right? would be nice if we could update it automatically as well in the `release` script

**Comment by lispyclouds** [comment]
yes, hence i wanted to know where and how the `release` file is used/called from.

**Comment by lispyclouds** [comment]
Added it now nonetheless.

**Review by ericdallo — APPROVED** [review]

**Comment by ericdallo** [comment]
The release file is manually run when we want a new release, we could do a git tag and leave a GH action do that as well, but it's almost the same, thank you!

**Pull request merged**